Packs/CommonScripts/Scripts/PrintErrorEntry/PrintErrorEntry_test.py

```diff
@@ -0,0 +1,17 @@
+import demistomock as demisto
+from CommonServerPython import * # noqa: F401
+
+
+def test_main(mocker):
+    from PrintErrorEntry import main
```

Comment:
```suggestion
    import PrintErrorEntry
```
And move it outside of the test_main method